### DISCUSSION TITLE
How to change the final bundle name?

### DISCUSSION THREAD

**Discussion by SamuelC0507** [tauri-apps/tauri#6417] [Q&A] [answered]
Hi guys, can we change the final bundle (e.g. the MSI file on Windows) name in any place? Like to use a different name other than the productName specified in tauri.conf.json, or don't include the version and language of the app?

I have no problem changing the name afterwards manually though, just wondering if there's already a place to configure this?

**Comment by FabianLars** [comment, marked as answer]
You have to rename it yourself. And to be honest i'm not sure if something like this will ever be implemented in the cli 🤔

**Reply by SamuelC0507** [reply]
Got it thanks!

**Reply by iDm1** [reply]
There is a problem. The "productName" is used also for the installer title and executable in the installer. But those are usually should be different.

Executable usually has a shortened name. But installer title should contain full name of the package, otherwise it looks ugly. So some kind of separate bundleName or executableName variable is still required.

**Reply by Deathrage** [reply]
I agree with @iDm1

**Reply by kurdin** [reply]
It would be nice to have something like `afterBuildCommand` hook similar to `beforeBuildCommand`

**Reply by kurdin** [reply]
@FabianLars on Mac, when you build an app, the bundler does not add `version number` and `platform name` to the final file name for updater in `release > bundle > macos`  
I have to manually rename  `PasteBar.app.tar.gz` and `PasteBar.app.tar.gz.sig` to `PasteBar_0.2.3_aarch64.app.tar.gz` and `PasteBar_0.2.3_aarch64.app.tar.gz.sig`. This does not happen when building for Windows.

**Comment by velocitysystems** [comment]
Created a feature request.
https://github.com/tauri-apps/tauri/issues/13999

**Comment by wojtekmach** [comment]
FWIW, for anyone already using `tauri-apps/tauri-action@v0`, it's a one line change and it's working great for us:

```diff
  uses: tauri-apps/tauri-action@v0
  with:
    ...
+   assetNamePattern: "[name]-[arch][ext]"
```
